### PR TITLE
Add copyright to Capable Care app [sc-7716]

### PR DESCRIPTION
**Pull request by lvancraen**
Story: https://app.shortcut.com/capable/story/7716/add-copyright-to-capable-care-app

Added copyright with new env var `REACT_APP_COMPANY_NAME`.

Screen Shot 2022-09-07 at 8 55 18 AM

### CODE REVIEW

**Review by ezekielp — APPROVED** [review]
👍

File: src/components/MobileContainer.js [hunk 1]
```diff
@@ -11,6 +11,12 @@ export default function MobileContainer({ children }) {
       navigator.userAgent
     );
 
+  const CopyrightFooter = (
+    <footer style={{ fontSize: "small", backgroundColor: "#fafafa", textAlign: "center" }}>
+      &copy; {new Date().getFullYear()} {process.env.REACT_APP_NAME}. All rights reserved.
```
Comment (LUMacKenn) [inline src/components/MobileContainer.js:16]:
```suggestion
      &copy; {new Date().getFullYear()} {process.env.REACT_APP_COMPANY_NAME}. All rights reserved.
```

Comment (LUMacKenn) [inline src/components/MobileContainer.js:16]:
```suggestion
      &copy; {new Date().getFullYear()} {process.env.REACT_APP_COMPANY_NAME}. All rights reserved.
```

File: src/components/MobileContainer.js [hunk 2]
```diff
@@ -23,6 +29,7 @@ export default function MobileContainer({ children }) {
         }}
       >
         {children}
+        {CopyrightFooter}
```
Comment (LUMacKenn) [inline src/components/MobileContainer.js:32]: Let's render this in both places as a JSX element

**Review by LUMacKenn — CHANGES_REQUESTED** [review]

**Review by lawyu89 — APPROVED** [review]

**Review by LUMacKenn — APPROVED** [review]

**Pull request merged**